src/Ledger/Utxow.lagda

```diff
@@ -186,4 +195,24 @@ pattern UTXOW⇒UTXO x = UTXOW-inductive⋯ _ _ _ _ _ _ _ _ x
 unquoteDecl UTXOW-inductive-premises =
   genPremises UTXOW-inductive-premises (quote UTXOW-inductive)
 \end{code}
-\end{NoConway}
+
+\href{https://github.com/cardano-foundation/CIPs/tree/master/CIP-0069}{CIP-0069}
+unifies the arguments given to all types of Plutus scripts currently available
+(spending, certifying, rewarding, minting) by removing the argument of a datum.
+This aims to address the mutual dependency issue (two validators that need to
+know each other's hash), which arises when designing dapps and is widely
+considered a substantial barrier to safe protocols and a considerable limitation
+on our design space.
+
+According to CIP-0069, every validator must take the redeemer and script
+context as arguments.  Datums are provided by either looking them up in the
+\texttt{ScriptContext} or extending the \texttt{Spending} constructor of
+\texttt{TxInfo} to carry (\texttt{TxOutRef}, \texttt{Datum}).
```

Comment:
This sentence is somewhat irrelevant since we're not changing how the datum is passed in this spec. However it may be worth mentioning why we don't have to change anything: Since the interface with Plutus is kept abstract in this specification, the changes to the representation of the script context which are part of CIP-69 are not included here. To provide a CIP-69 conformant implementation of Plutus to this specification an additional step processing the `List Data` argument we provide is required.